apps/microtvm/arduino/template_project/test_microtvm_api_server.py

```diff
@@ -0,0 +1,38 @@
+from unittest.mock import patch
```

Comment:
i can't remember if we talked about this--I think we did...you could put this in `tests/micro/arduino` and save the remove-test-case logic. i'd suggest we either do that or at least put it in `apps/microtvm/arduino/tests`. however, there's precedent for testing `apps/` (and other dirs, like `python` and `src`) in `tests/`, so i don't think it's a big deal to put this in `tests/micro/arduino`.

Reply:
You initially suggested putting it right next to `microtvm_api_server.py`, but I agree moving it to `apps` is cleaner. Done.

Reply:
We discussed this over Slack, and ended up placing it in `apps/microtvm/arduino/template_project/tests`.